packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -318,6 +318,8 @@ class DeepLinksController extends DisposableController {
       return false;
     }
     localFingerprint.value = fingerprint;
+    // If the local fingerprint is updated, re-generate asset link file.
+    unawaited(_generateAssetLinks());
```

Comment:
should we check that the previous value of `localFingerprint.value` does not match `fingerprint` before doing this? that's what the comment makes it sound like we should do.